### PR TITLE
Update distillation

### PR DESCRIPTION
**Pull request by jinPrelude**
1. Change method for gathering distillation data
2. Change method for loading distillation data

### CODE REVIEW

File: rl_algorithms/common/buffer/distillation_buffer.py [hunk 1]
```diff
@@ -81,7 +82,7 @@ class DistillationDataset(Dataset):
 
     """
 
-    def __init__(self, buffer_path: str, buffer_size: int):
+    def __init__(self, buffer_path: str):
```
Comment (MrSyee) [inline rl_algorithms/common/buffer/distillation_buffer.py:85]: str -> list

File: rl_algorithms/dqn/agent.py [hunk 2]
```diff
@@ -93,6 +95,15 @@ def __init__(
             self.min_epsilon = hyper_params.min_epsilon
             self.epsilon = hyper_params.max_epsilon
 
+        if self.args.save_distillation_data:
```
Comment (MrSyee) [inline rl_algorithms/dqn/agent.py:98]: should remove

File: rl_algorithms/dqn/agent.py [hunk 3]
```diff
@@ -199,6 +210,23 @@ def pretrain(self):
         """Pretraining steps."""
         pass
 
+    def _save_distillation_data(self, state):
```
Comment (MrSyee) [inline rl_algorithms/dqn/agent.py:213]: should remove

**Comment by MrSyee** [comment]
You should change the commit message and make new PR.

**Comment by jinPrelude** [comment]
Close PR since it's contains unnecessary contents.

**Pull request closed without merging**